src/utils.jl

```diff
@@ -125,6 +125,18 @@ function cov(x::AbstractVector{Measurement{T}}) where T
     return Symmetric(covariance_matrix, :L)
 end
 
+"""
+    Measurements.cov(a::Measurement, b::Measurement)
+
+Returns the (scalar) covariance between `Measurment`s `a` and `b`.
+"""
+function cov(a::Measurement{T}, b::Measurement{T}) where {T}
+    overlap = keys(a.der) ∩ keys(b.der)
+    return isempty(overlap) ? zero(T) : sum(overlap) do var
+        a.der[var] * b.der[var] * var[2]^2
+    end
+end
```

Comment:
Can the `cov` method just above be implemented calling this method?  Similarly for `corr` below.

Reply:
Looks easy enough for `cov`, I'll give it a try and then at least there's no code duplication